core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -366,41 +346,6 @@ public String getPluralName() {
         return result;
     }
 
-    private List<ModelField> sortModelFields() {
```

Comment:
The sorting currently happens in the `FieldFinder` utility, and this is never used.